scripts/pre-release.sh

```diff
@@ -36,67 +38,51 @@ fi
 echo "Proceeding..."
 
 set -e
-TEMP_WORKING_DIR=~/tmp/narayana/$CURRENT/sources/
+TEMP_WORKING_DIR=~/tmp/narayana/$CURRENT/
 mkdir -p $TEMP_WORKING_DIR
 cd $TEMP_WORKING_DIR || fatal
 
-for REPO in documentation quickstart performance narayana
```

Comment:
Why did you remove documentation quickstart and performance @jmfinelli ? I think we should keep tagging the version for those project too.